src/components/groups/GroupCard.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+interface CardProps {
+  groupName: string;
+  memberCount: number;
+  capacity: number;
+  handleCardClick: () => void;
+}
+
+const GroupCard: React.FC<CardProps> = ({
+  groupName,
+  memberCount,
+  capacity,
+  handleCardClick,
+}) => {
+  
+
+  return (
+    <div
+      onClick={handleCardClick}
+      className="flex items-center p-4 bg-white max-w-xs w-60 shadow-sm rounded-xl border-l border-r border-b"
+    >
+      <div className="card-item">
```

Comment:
is `card-item` a custom class? If it not doing anything please remove it from the code.